### PR TITLE
testpage-jsx_profile card for viewing

### PR DESCRIPTION
**Pull request by Lukeski91**
added a ui for viewing of profile_profile card

### CODE REVIEW

File: client/src/pages/testpage.jsx [hunk 1]
```diff
@@ -1,13 +1,353 @@
...
+            <button>
+              <i
+                className="fa-solid fa-arrow-up-from-bracket"
+                style={{ color: "#fff", width: "20px" }}
+              ></i>
+            </button>
+          </div>
+
+          <div className="info">
+            {address}
```
Comment (IrrationalCucumber) [inline client/src/pages/testpage.jsx:190]: Need to be editable

File: client/src/pages/testpage.jsx [hunk 2]
```diff
@@ -1,13 +1,353 @@
...
+            type="number"
+            name="age"
+            // className="display-data1"
+            placeholder="Age"
+            value={age}
+            //onChange={props.handleChange}
+            min={1}
+            max={99}
+          ></input>
+          <input
```
Comment (IrrationalCucumber) [inline client/src/pages/testpage.jsx:281]: Adjust style because of additional input data

File: client/src/pages/testpage.jsx [hunk 3]
```diff
@@ -1,13 +1,353 @@
...
+            <img
+              //profile file path in backend
+              src={`http://localhost:8800/images/profile/${profileImage}`}
+              alt="ProfPic"
+            />
+          ) : (
+            <img src="/images/employer.png" alt="Profile Picture" />
+          )}
+
+          <div className="upload-container">
```
Comment (IrrationalCucumber) [inline client/src/pages/testpage.jsx:174]: Add style to this for uploading photo

File: client/src/pages/testpage.jsx [hunk 4]
```diff
@@ -1,13 +1,353 @@
...
+          </div>
+
+          <div className="info">
+            {address}
+            <br />
+            {email}
+            <br />
+            {contact}
+          </div>
+          <textarea
```
Comment (IrrationalCucumber) [inline client/src/pages/testpage.jsx:196]: Changed to textarea instead of list for user Description

**Review by IrrationalCucumber — CHANGES_REQUESTED** [review]

**Comment by IrrationalCucumber** [comment]
image
image
Applied profile design with adjustment

**Review by Mamonde02 — APPROVED** [review]

**Review by IrrationalCucumber — APPROVED** [review]

**Pull request merged**